packages/suite/src/views/guide/Feedback.tsx

```diff
@@ -187,6 +195,45 @@ const Feedback = ({ type }: Props) => {
         setView('GUIDE_DEFAULT');
     }, [device, firmwareType, type, setView, sendFeedback, description, category, rating?.id]);
 
+    const feedbackCategories: FeedbackCategory[] = [
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_DASHBOARD" />, value: 'dashboard' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_ACCOUNT" />, value: 'account' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_SETTINGS" />, value: 'settings' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_SEND" />, value: 'send' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_RECEIVE" />, value: 'receive' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_TRADE" />, value: 'trade' },
+        { label: <Translation id="TR_FEEDBACK_CATEGORY_OTHER" />, value: 'other' },
+    ];
+
+    // Router apps does not match 1:1 to Feedback Categories
+    const getDefaultCategory = (): FeedbackCategory | undefined => {
+        const { app, route } = router;
+        const routePattern = route?.pattern || '';
+
+        if (routePattern.startsWith('/accounts/coinmarket')) {
+            return feedbackCategories.find(item => item.value === 'trade');
+        }
+        if (routePattern.startsWith('/accounts/send')) {
+            return feedbackCategories.find(item => item.value === 'send');
+        }
+        if (routePattern.startsWith('/accounts/receive')) {
+            return feedbackCategories.find(item => item.value === 'receive');
+        }
+
+        switch (app) {
+            case 'dashboard':
+                return feedbackCategories.find(item => item.value === 'dashboard');
+            case 'wallet':
+                return feedbackCategories.find(item => item.value === 'account');
+            case 'settings':
+                return feedbackCategories.find(item => item.value === 'settings');
+            default:
+                return undefined;
```

Comment:
This means that if the user is on some unhandled 'app' there will be no category selected.

With the @tomexx's approach the `other` category was selected in that case.

I think your solution yields a slightly better UX but I just wanted to point out this detail.

Reply:
Yes exactly, I think that it is better not to fill anything if we are not sure what a user wants to report. Because when there is nothing selected, a user has to think, what category is good for his issue. 
If "other" is selected right away, a user will not change it and we will have issues without a category and waste time on it.